commit af770a1558c4415cd4b74cf7bf30e09be7b77a30
Author: Matt Ranostay <matt.ranostay@konsulko.com>
Date:   Sun Dec 30 19:07:01 2018 -0800

    iio: chemical: atlas-ph-sensor: correct IIO_TEMP values to millicelsius
    
    commit 0808831dc62e90023ad14ff8da4804c7846e904b upstream.
    
    IIO_TEMP scale value for temperature was incorrect and not in millicelsius
    as required by the ABI documentation.
    
    Signed-off-by: Matt Ranostay <matt.ranostay@konsulko.com>
    Fixes: 27dec00ecf2d (iio: chemical: add Atlas pH-SM sensor support)
    Cc: <stable@vger.kernel.org>
    Signed-off-by: Jonathan Cameron <Jonathan.Cameron@huawei.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/iio/chemical/atlas-ph-sensor.c b/drivers/iio/chemical/atlas-ph-sensor.c
index a406ad31b096..3a20cb5d9bff 100644
--- a/drivers/iio/chemical/atlas-ph-sensor.c
+++ b/drivers/iio/chemical/atlas-ph-sensor.c
@@ -444,9 +444,8 @@ static int atlas_read_raw(struct iio_dev *indio_dev,
 	case IIO_CHAN_INFO_SCALE:
 		switch (chan->type) {
 		case IIO_TEMP:
-			*val = 1; /* 0.01 */
-			*val2 = 100;
-			break;
+			*val = 10;
+			return IIO_VAL_INT;
 		case IIO_PH:
 			*val = 1; /* 0.001 */
 			*val2 = 1000;
@@ -477,7 +476,7 @@ static int atlas_write_raw(struct iio_dev *indio_dev,
 			   int val, int val2, long mask)
 {
 	struct atlas_data *data = iio_priv(indio_dev);
-	__be32 reg = cpu_to_be32(val);
+	__be32 reg = cpu_to_be32(val / 10);
 
 	if (val2 != 0 || val < 0 || val > 20000)
 		return -EINVAL;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1315_af770a1558c4415cd4b74cf7bf30e09be7b77a30.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1315_af770a1558c4415cd4b74cf7bf30e09be7b77a30.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
